Avoids location.hh build dependency error. This can be removed when 0.25 is released.

--- a/src/output/CMakeLists.txt
+++ b/src/output/CMakeLists.txt
@@ -3,4 +3,5 @@ add_library(output STATIC
   output.cpp
   text.cpp
   )
+add_dependencies(output parser)
 target_link_libraries(output util)

